third_party/docusaurus-search-local/src/server/utils/processDocInfos.ts

```diff
@@ -60,8 +62,7 @@ export function processDocInfos(
       }
       if (
         indexDocs &&
-        docsRouteBasePath.some((basePath) => urlMatchesPrefix(route, basePath)) &&
-        isStableVersion(url)
+        docsRouteBasePath.some((basePath) => isSameOrSubRoute(route, basePath)) && isStableVersion(url)
```

Comment:
the whole third_party folder content was automatically updated except modified with the isStableVersion change.